docs/current/sdk/elixir/756758-get-started.md

```diff
@@ -0,0 +1,117 @@
+
+```elixir
+def deps do
+  [
+    {:dagger, "~> 0.8", only: [:dev, :test]}
+  ]
+end
+```
+
+Run `mix deps.get` to fetch the Elixir SDK from Hex.pm:
+
+```shell
+mix deps.get
+```
+
+## Step 3: Create the Mix task
+
+Create a new module and Mix task to test the project at `lib/mix/tasks/a_project.test.ex`:
+
+```elixir file=snippets/get-started/step3/a_project.test.ex
+```
+
+This module performs the following operations:
+
+- It starts all applications related to the Dagger Elixir SDK with `Application.ensure_all_started(:dagger)`.
+- It creates a Dagger client with `Dagger.connect!/1`.
+- It uses the client's `Dagger.Client.host/1` and `Dagger.Host.directory/3` to obtain a reference to the host directory. It additionally uses the `exclude` filter in `Dagger.Host.directory/3` to filter unwanted files and directories.
+- It uses the client's `Dagger.Client.container/2` and `Dagger.Container.from/2` to initialize a new container from the `hexpm/elixir:1.15.4-erlang-25.3.2.5-ubuntu-bionic-20230126` base image.
+- It uses `Dagger.Container.with_mounted_directory/3` to mount the project's source files into the container.
+- It uses `Dagger.Container.with_exec/3` to define the commands to be executed in the container - in this case, commands such as `mix deps get`, which downloads dependencies and `mix test`, which runs unit tests. Each invocation of `with_exec` returns a revised `Container` with the results of command execution.
+- It uses `Dagger.Sync.sync/1` to force command execution.
+- It uses `Dagger.close/1` to close the client connection.
+
+Run the Mix task by executing the command below from the project directory:
+
+```shell
+dagger run mix a_project.test
+```
+
+The `dagger run` command executes the specified command in a Dagger session and displays live progress. Here is an example of the output:
+
+```shell
+Tests succeeded!
+```
+
+## Step 4: Test against multiple Elixir and Erlang/OTP versions
+
+Now that the Elixir CI tool can test the application against a specified Elixir and Erlang/OTP version, the next step is to extend it for multiple Elixir and Erlang/OTP versions.
+
+Replace the `lib/mix/tasks/a_project.test.ex` file from the previous step with the version below:
+
+```elixir file=snippets/get-started/step4/a_project.test.ex
+```
+
+This version has additional support for testing and building against multiple Elixir and Erlang/OTP versions:
+
+- It defines the test matrix, consisting of a list of Elixir and Erlang/OTP version pairs.
+- It uses `Task.async_stream/3` to run tests against each version pair concurrently.
+
+Run the tool again by executing the command below:
+
+```shell
+dagger run mix a_project.test
+```
+
+The tool tests the application, logging its operations to the console as it works. If all tests pass, it displays the final output below:
+
+```shell
+Starting tests for Erlang 1.14.5 and Erlang OTP 25.3.2.5
+Tests for Python Erlang 1.14.5 and Erlang OTP 25.3.2.5 succeeded!
+TODO
```

Comment:
@wingyplus following a structure similar to that the Python SDK get started, can you please update your code to display this type of output at the end? Please then also update this TODO block with the revised output

Reply:
Updated!